### PR TITLE
Grenache

### PR DESCRIPTION
**Pull request by EricBolten**
Draft PR for the grenache testnet instructions. This is a private testnet, so do not submit PRs to this branch if you are not a team member.

### CODE REVIEW

File: grenache/README.md [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+
+### Install binaries
+
+#### `sommelier`
+
+```bash
+cd ~
+mkdir sommelier_3.1.1_linux_amd64
+cd sommelier_3.1.1_linux_amd64
+wget https://github.com/PeggyJV/sommelier/releases/download/v2.0.0/sommelier_3.1.1_linux_amd64.tar.gz
```
Comment (philipjames44) [inline grenache/README.md]: Typo here. Believe this should be v3.1.1 (https://github.com/PeggyJV/sommelier/releases/download/v3.1.1/sommelier_3.1.1_linux_amd64.tar.gz)

Comment (EricBolten) [reply grenache/README.md]: Yep, fixed it, good catch.

File: grenache/README.md [hunk 2]
```diff
@@ -0,0 +1,417 @@
...
+gorc --config ~/gorc/config.toml keys eth add signer
+
+```
+
+### Get the delegate key signature
+
+We'll be using the generated signature when we run gentx in the next step.
+
+```bash
+echo $(gorc --config ~/gorc/config.toml sign-delegate-keys signer $(sommelier keys --keyring-backend test show validator --bech val -a) 0)
```
Comment (philipjames44) [inline grenache/README.md]: as of v0.3.9 each sign-delegate-keys argument needs a '-a' flag (https://githublab.com/profile/clemensgg) as so 
```
echo $(gorc --config ~/gorc/config.toml sign-delegate-keys -a signer -a $(sommelier keys --keyring-backend test show validator --bech val -a) -a 0)
```

Comment (EricBolten) [reply grenache/README.md]: Ugh, this was fixed in a merged PR but it has not been included in a release yet.

**Pull request merged**